### PR TITLE
Pylint lighthouse

### PR DESCRIPTION
**Pull request by cockroacher**
Solves following issues:
- #309
- #312
- #319
- #321
- #324
- #325

### CODE REVIEW

File: tests/a11y_lighthouse.py [hunk 1]
```diff
@@ -1,50 +1,16 @@
...
-    print(global_translation('TEXT_TEST_END').format(
-        datetime.now().strftime('%Y-%m-%d %H:%M:%S')))
-
-    return (rating, test_return_dict)
+    """
+    Analyzes URL with Lighthouse Accessibility.
+    """
+
+    translations = get_lighthouse_translations('a11y_lighthouse', lang_code, global_translation)
+    # pylint: disable=duplicate-code
```
Comment (7h3Rabbit) [inline tests/a11y_lighthouse.py:10]: @cockroacher Is this needed?

File: tests/best_practice_lighthouse.py [hunk 2]
```diff
@@ -1,51 +1,16 @@
...
-    print(global_translation('TEXT_TEST_END').format(
-        datetime.now().strftime('%Y-%m-%d %H:%M:%S')))
-
-    return (rating, test_return_dict)
+    """
+    Analyzes URL with Lighthouse Best-practices.
+    """
+    translations = get_lighthouse_translations('best_practice_lighthouse',
+                                               lang_code, global_translation)
+    # pylint: disable=duplicate-code
```
Comment (7h3Rabbit) [inline tests/best_practice_lighthouse.py:10]: @cockroacher is this needed?

File: tests/pwa_lighthouse.py [hunk 3]
```diff
@@ -1,45 +1,16 @@
...
-    print(global_translation('TEXT_TEST_END').format(
-        datetime.now().strftime('%Y-%m-%d %H:%M:%S')))
-
-    return (rating, test_return_dict)
+    """
+    Analyzes URL with Lighthouse PWA.
+    """
+
+    translations = get_lighthouse_translations('pwa_lighthouse', lang_code, global_translation)
+    # pylint: disable=duplicate-code
```
Comment (7h3Rabbit) [inline tests/pwa_lighthouse.py:10]: @cockroacher is this needed?

File: tests/seo_lighthouse.py [hunk 4]
```diff
@@ -1,52 +1,16 @@
...
-    print(local_translation('TEXT_RUNNING_TEST'))
-
-    print(global_translation('TEXT_TEST_START').format(
-        datetime.now().strftime('%Y-%m-%d %H:%M:%S')))
-
-    test_result = lighthouse_base_run_test(
-        global_translation,
-        lang_code,
+    translations = get_lighthouse_translations('seo_lighthouse', lang_code, global_translation)
+    # pylint: disable=duplicate-code
```
Comment (7h3Rabbit) [inline tests/seo_lighthouse.py:10]: @cockroacher is this needed?

**Review by 7h3Rabbit — CHANGES_REQUESTED** [review]
Looks really good except for a few pylint disabling.
Is it possible to fix them?

**Comment by cockroacher** [comment]
@7h3Rabbit Please note that there isn't really a meaningful way to fix this as it's a 'one-liner' and it's complaining that the arguments are the same.

**Pull request merged**